QModManager/Patching/ManifestValidator.cs

```diff
@@ -14,6 +14,14 @@ internal class ManifestValidator
     {
         internal static readonly Regex VersionRegex = new Regex(@"(((\d+)\.?)+)");
 
+        internal static readonly Dictionary<string, ModStatus> Exceptions = new Dictionary<string, ModStatus>()
```

Comment:
Can `Exceptions` be renamed to something more informative?  
Maybe `ProhibitedModIds`?